web/src/ui.rs

```diff
@@ -283,12 +256,8 @@ impl UiBackend for WebUiBackend {
             let mut dialog = AsyncFileDialog::new();
 
             for filter in filters {
-                let window = web_sys::window().expect("window()");
-                let navigator = window.navigator();
-                let platform = navigator.platform().expect("navigator.platform");
-
-                if platform.contains("Mac") && filter.mac_type.is_some() {
-                    let mac_type = filter.mac_type.expect("Cant fail");
+                if std::env::consts::OS == "macos" && filter.mac_type.is_some() {
```

Comment:
Just a question: Is `std::env::consts::OS` set correctly (to `"macos"`) even in WebAssembly?